### PR TITLE
Symfony 3 compatibility

### PR DESCRIPTION
**Pull request by citadev**
Hi,

First thank you for this project, I'm using it for quite a while now.

I need to use Symfony 3 on a project so I just made a few changes in order to use your bundle : 
- Changed the requirement for symfony/process from "2.*" to "2.*|3.*"  in composer.json 
- Replaced $proc->setStdin by $proc->setInput in StringInput.php

I ran your phpunit tests (using docker on my laptop), and all is green.

Sorry if I made a few strange commits, I'm new to contributing and github...

Regards,

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
-	"name" : "goetas/apache-fop",
-	"description" : "Symfony2 Apache FOP Bundle",
+	"name" : "citadev/apache-fop",
```
Comment (goetas) [inline composer.json]: this should stay `goetas`

**Comment by citadev** [comment]
Ok sorry, I didn't know that.
Thank you for the link, I had exactly the same problem, and that is why I renamed the package.

So, do I have to do something (revert the package name) ? Or is it good ?

**Comment by goetas** [comment]
yes, revert please

**Comment by goetas** [comment]
if you are interested here http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html you can find a guide on how to squash commits

**Comment by citadev** [comment]
thanks for the guide, I will check that.

**Pull request merged**